src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-list_of_link_teasers.js

```diff
@@ -1,11 +1,17 @@
-const contentModelType = 'paragraph-list_of_link_teasers';
-
 module.exports = {
   type: 'object',
-  baseType: {
-    enum: ['paragraph'],
-  },
-  contentModelType: {
-    enum: [contentModelType],
+  properties: {
+    contentModelType: { enum: ['paragraph-list_of_link_teasers'] },
+    entity: {
+      type: 'object',
+      properties: {
+        entityType: { enum: ['paragraph'] },
+        entityBundle: { enum: ['list_of_link_teasers'] },
+        fieldTitle: { type: ['string', 'null'] },
+        fieldVaParagraphs: { type: 'array' },
```

Comment:
```suggestion
        fieldVaParagraphs: {
          type: 'array',
          items: { $ref: 'transformed/paragraph-link_teaser' },
        },
```
Alternatively, the `items` can have a `$ref: 'Paragraph'`, but I think that's probably a little broad for this particular one.